### DISCUSSION TITLE
Problem with new updates of nuxt 3

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17824] [Questions]
Today i update my project with nuxt 3 to version `3.0.0-27457855.7c0b914`
And now i have errors when i trying to start project:
`[Vue warn]: Failed to resolve component: LazyBitrixFormsRequestCall
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.`

I have looked at the new Breaking changes, but I don't understand how to solve my problem
in my `nuxt.config.ts` i have: 
```ts
export default defineNuxtConfig({
	publicRuntimeConfig: {},
	css: ['~/assets/css/main.css', '~/assets/scss/main.scss'],
	app: {
		buildAssetsDir: 'assets/',
	},
	components: {
		global: true,
		dirs: ['~/components'],
	},
	router: {
		prefetchLinks: false,
	},
	nitro: {
		output: {
			dir: 'build',
		},
	},
	srcDir: 'src',
	build: {
		splitChunks: {
			pages: true,
			layouts: true,
		},
	},
	buildModules: ['@nuxtjs/tailwindcss', '@vueuse/nuxt'],
	modern: true,
	tailwindcss: {
		configPath: '~~/tailwind.config.js',
		viewer: false,
	},
	meta: {
		charset: 'utf-8',
		viewport: 'width=device-width, initial-scale=1, maximum-scale=1',
		htmlAttrs: {
			lang: 'ru',
		},
		title: 'SANTREK',
		link: [
			{
				rel: 'icon',
				href: '/favicon.svg',
				type: 'image/svg+xml',
			},
		],
	},
	vite: {
		plugins: [svgLoader()],
	},
});
```

**Comment by lucassimines** [comment]
I'm having the same problem, and another one that shows the following message:

```
runtime-dom.esm-bundler.js:10 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'insertBefore')
```

I'm currently using the following version that is working `nuxt3@3.0.0-27457579.3f91d83`

**Reply by little3201** [reply]
3.0.0-27459267.f69126e too
